Milestone 0- docs(collabortion): update communication.md

AmmarIbrahimTech opened the pull request:
---
Name: updated the communication file
about: updated the communication file according to what we discussed in slack
---

<!--
  make this PR easy to find:

  - assign and add reviewers
  - add any helpful labels
  - connect it to a milestone (if necessary)
  - link it with an issue (if necessary)
-->

<!-- A clear and concise description of what the Pull Request is about. -->

# Updated communication file:
<!--
    this template is for inspiration, feel free to change it however you like!

    Careful! be sure to protect your privacy when filling out this document
        everything you write here will be public
        so share only what you are comfortable sharing online
        you can share the rest in confidence with your group by another channel
-->

# Communication

---

## Communication Schedule

| Day       | How        | The topic of discussion                         |
|---------- |:----------:|----------------------------------------------- |
| As needed    | Slack      | General communication and coordination         |
| Weekly    | Google Meet| Team meetings and check-ins                    |
| As needed | GitHub     | Task-related discussions (Issues, PR reviews)  |

## Communication Channels

How often will we get in touch on each channel, and what we will discuss there:

- **Issues and Pull Requests**: Used for task-specific discussions and reviewing submitted code. Helps in learning how to manage GitHub collaboration. We should establish clear norms on when to close issues and accept PRs.
- **Slack**: Preferred for daily communication, quick questions, and coordination.
- **Video Calls**: Held on Google Meet. Used for meetings. Weekly or as needed.

---

## Availability

### Availability for calling/messaging

| Day    | Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday |
|--------|:------:|:-------:|:---------:|:--------:|:------:|:--------:|:------:|
| Ammar  |   12–3PM   |   12–3PM    |    12–3PM     |    12–3PM    |   ❌   |    12–3PM    |   12–3PM   |
| Banu   | 12–4PM |  2–4PM  |   2–4PM   |  12–4PM  | 12–4PM | 12–4PM   | 12–4PM |
| Amro   |   ❌   |  9–3PM  |    ❌     |  9–3PM   |  9–3PM |  9–3PM   |  9–3PM |
| Azza   | 3–5AM, 12–2PM | 3–5AM, 12–2PM | 3–5AM, 12–2PM | 3–5AM, 12–2PM | ❌ | 5–7AM | 3–5AM, 12–2PM |
|Muhammad| 12–2PM | 12–2PM | 12–2PM | 12–2PM | ❌ | ❌  | 12–2PM |

> Note: Times are in EST. Availability may vary slightly due to personal commitments.

### How many hours everyone has per day

- Ammar: _2h_
- Banu: _2–3h_
- Amro: _2–3h_
- Azza: _2-3h and Emergency_
- Muhammamd: 2h

---

## Asking for Help

There's a fine line between confidently learning from your mistakes, and stubbornly getting nowhere. Here is a general guide for when to ask for help based on how long you've been stuck on the same problem:

1. _0 → 30 min_: Try on your own  
2. _30 → 60 min_: Ask your group for help  
3. _60+ min_: Tag your coaches in Slack or GitHub

---
#Checklist

## General Checks

- [x] the branch is up to date with `main`
- [x] the code works when pulled and run locally
- [x] All CI checks pass (or at least discussed)
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

<!-- markdown-specific checks -->

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work

AzzaOmer1 commented:
great work

Elshikh-Amro reviewed (approved):
great work man!!!!!!

doctorbanu commented on collaboration/communication.md:
```diff
@@ -13,42 +13,50 @@
+- Banu: _2–3h_
+- Amro: _2–3h_
+- Azza: _2-3h and Emergency_
+- Muhammamd: 2h
```
I guess there might be a very minor typo here. (the last 'm' is extra). Just something that caught my eye 😅

AmmarIbrahimTech replied:
Good catch 👏

doctorbanu reviewed (approved):
Nicely done Ammar. I am approving it. It's just I believe there is just a minor typo on line 52, there is an extra 'm' at the and of Muhammad's name. Maybe you might want to edit that line. Thank you for your efforts.

AmmarIbrahimTech commented:
Error: collaboration/communication.md:26:81 MD013/line-length Line length [Expected: 80; Actual: 222]
Error: collaboration/communication.md:3[8](https://github.com/MIT-Emerging-Talent/ET6-CDSP-group-19-repo/actions/runs/15379416872/job/43268231933?pr=2#step:4:9):81 MD013/line-length Line length [Expected: 80; Actual: 104]
Error: collaboration/communication.md:41:81 MD013/line-length Line length [Expected: 80; Actual: 102]
Error: collaboration/communication.md:58:81 MD013/line-length Line length [Expected: 80; Actual: 206]

I had to keep these errors to keep the table structure organized.

Elshikh-Amro commented:
You need to solve this issues because you are working in this file my solution won't work

doctorbanu commented:
> Error: collaboration/communication.md:26:81 MD013/line-length Line length [Expected: 80; Actual: 222]
> 
> Error: collaboration/communication.md:3[8](https://github.com/MIT-Emerging-Talent/ET6-CDSP-group-19-repo/actions/runs/15379416872/job/43268231933?pr=2#step:4:9):81 MD013/line-length Line length [Expected: 80; Actual: 104]
> 
> Error: collaboration/communication.md:41:81 MD013/line-length Line length [Expected: 80; Actual: 102]
> 
> Error: collaboration/communication.md:58:81 MD013/line-length Line length [Expected: 80; Actual: 206]
> 
> 
> 
> I had to keep these errors to keep the table structure organized.

So, I asked GitHub Copilot about this and it suggested the following solution, can you please try this? Hope it works🙏

"This is a common problem when using wide tables in Markdown docs, especially when CI checks enforce line length limits (e.g., 80 or 120 characters per line) and formatting rules (like those enforced by linters such as markdownlint, mdl, or Prettier).

Here is a practical way to solve or work around these issues while keeping all the table information:

Disable Line Length Checks for Tables

Most Markdown linters support disabling specific rules for certain sections.
You can add linter "ignore" comments above and below your table to skip line length enforcement just for that table."

markdownlint example:

<!-- markdownlint-disable MD013 -->
| Column 1 | Column 2 | Column 3 | A very long column header that exceeds the limit |
|----------|----------|----------|-------------------------------------------------|
| ...      | ...      | ...      | ...                                             |
<!-- markdownlint-enable MD013 -->

AmmarIbrahimTech commented:
@doctorbanu , I tried your solution and it worked perfectly. Thank you! I learned something new 🥳

@Elshikh-Amro I’ve fixed all the line length issues in the communication file. I left the table as-is to avoid breaking it into smaller parts, but I added a comment explaining why I ignored those lines.

doctorbanu reviewed (dismissed):
Yayyy 🥳 we managed to defeat markdown line length error 🙌 nice job Ammar

AzzaOmer1 reviewed (approved):
good work
